state/protocol/badger/snapshot_test.go

```diff
@@ -272,45 +272,51 @@ func TestSealingSegment(t *testing.T) {
 
 	// test sealing segment where the segment blocks contain seals for
 	// ancestor blocks prior to the sealing segment
-	// ROOT -> B1 -> B2 -> B3(S1) -> B4(S2)
+	// ROOT -> B1 -> B2(R1) -> B3(R2, S1) -> B4 -> B5(S2)
```

Comment:
Hm, to me it seems that there is no gap between R1 and S1, it should be something like this:
`R -> B1 -> B2(R1) -> B3 -> B3(R2, S1) -> B4 -> B5(S2)`